docs/reference/modules/cluster/shards_allocation.asciidoc

```diff
@@ -177,19 +206,25 @@ estimated number of indexing threads needed by the shard. Defaults to `10.0f`.
 Raising this value increases the tendency of {es} to equalize the total write
 load across nodes ahead of the other balancing variables.
 
-`cluster.routing.allocation.balance.threshold`::
-(float, <<dynamic-cluster-setting,Dynamic>>)
-The minimum improvement in weight which triggers a rebalancing shard movement.
-Defaults to `1.0f`. Raising this value will cause {es} to stop rebalancing
-shards sooner, leaving the cluster in a more unbalanced state.
-
 [NOTE]
 ====
-* It is not recommended to adjust the values of the heuristics settings. The
-default values are generally good, and although different values may improve
-the current balance, it is possible that they create problems in the future
-if the cluster or workload changes.
+* If you have a large cluster, it may be unnecessary to keep it in
+a perfectly balanced state at all times. It is less resource-intensive for the
+cluster to operate in a somewhat unbalanced state rather than to perform all
+the shard movements needed to achieve the perfect balance. If so, increase the
+value of `cluster.routing.allocation.balance.threshold` to define the
+acceptable imbalance between nodes. For instance, if you have an average of 500
+shards per node and can accept a difference of 5% (25 shards) between nodes,
+set `cluster.routing.allocation.balance.threshold` to `25`.
```

Comment:
I think it is more complex than that assuming ingest and disk usage forecast. Should this section only mention increasing this value would lead to less "balanced" cluster?

Reply:
My experience from conversations with users is that they do not have any intuition about what this threshold value means and therefore no way to know what an appropriate value might be. Just saying "increase the threshold" here would be no use without some quantification: some users will choose a conservative increase like `2` and see no benefit while others will go wild and choose `100000` with weird consequences.

I inserted the word "typical" in f43c4bd015b, does that help?